Bookish.Web/Views/Home/Catalogue.cshtml

```diff
@@ -13,35 +15,37 @@ else
 {
     <form asp-action="Catalogue">
         <input type="text" name="filter"/>
-        <input type="submit" value="Search"/>
+        <input type="submit" value="Search" />
     </form>
-    <table class="table">
-        <tr>
-            <th>
-                Title
-            </th>
-            <th>
-                Author
-            </th>
-            <th>
-                ISBN
-            </th>
-            <th>
-                Total copies
-            </th>
-            <th>
-                Copies available
-            </th>
-        </tr>
-        @foreach (var book in Model.Catalogue)
-        {
+    <div>
+        <table class="table">
```

Comment:
I would give it a more descriptive class name, e.g. `book-table` 🙂

Reply:
The table class was taken from bootstrap, should I define a new class with the same properties and assign that?

Reply:
Yes please. You can give an element several classes so feel free to give it the bootstrap table class as well 🙂